pkg/datapath/linux/requirements.go

```diff
@@ -139,10 +139,9 @@ func CheckMinRequirements() {
 		}
 		if err := probeManager.SystemConfigProbes(); err != nil {
 			errMsg := "BPF system config check: NOT OK."
-			// TODO(brb) warn after GH#14314 has been resolved
-			if !errors.Is(err, probes.ErrKernelConfigNotFound) {
-				log.WithError(err).Warn(errMsg)
-			}
+			// TODO(vincentmli): revisit log when GH#14314 has been resolved
+			// Warn missing required kernel config option
```

Comment:
This todo was specifically about ignoring the error, so I don't think it's necessary any more?